### PR TITLE
Fixup readme

### PR DESCRIPTION
**Pull request by weskamm**
Fixup misleading description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,12 +50,8 @@ If you want to contribute, you can build the project like this:
...
 
-and then either
+and then
 
-* `npm run build:dev`
-
-in order to get a development build or
-
-* `npm run build:dist`
+* `npm run build`
```
Comment (KaiVolland) [inline README.md:55]: Shouldn't this be `npm run start`?

**Comment by KaiVolland** [comment]
I guess `npm run build:dev` is replaced with `npm start` right? We might want to add this to the README.

**Comment by weskamm** [comment]
`npm start` seems to be broken and i dont see the usecase for it here. I can remove it if you agree @KaiVolland

**Comment by KaiVolland** [comment]
Ok. We should have a look at the setup. Check why therea are two tsconfig files and if we don't need webpack anymore we should remove it. But first we can merge this i guess. :)

**Pull request merged**